.github/CONTRIBUTING.md

```diff
@@ -118,47 +121,67 @@ var/path_type = /obj/item/weapon/baseball_bat
 var/path_type = "/obj/item/weapon/baseball_bat"
 ```
 
-### Tabs not spaces
+### Tabs, not spaces
 You must use tabs to indent your code, NOT SPACES.
 
 (You may use spaces to align something, but you should tab to the block level first, then add the remaining spaces)
 
-### No Hacky code
-Hacky code, such as adding specific checks, is highly discouraged and only allowed when there is ***no*** other option. (Protip: 'I couldn't immediately think of a proper way so thus there must be no other option' is not gonna cut it here )
+### No hacky code
+Hacky code, such as adding specific checks, is highly discouraged and only allowed when there is ***no*** other option. (Protip: 'I couldn't immediately think of a proper way so thus there must be no other option' is not gonna cut it here! If you can't think of anything else, say that outright and admit that you need help with it. Maintainers exist for exactly that reason.)
 
-You can avoid hacky code by using object oriented methodologies, such as overriding a function (called procs in DM) or sectioning code into functions and then overriding them as required.
+You can avoid hacky code by using object-oriented methodologies, such as overriding a function (called "procs" in DM) or sectioning code into functions and then overriding them as required.
 
-### No duplicated code.
-Copying code from one place to another maybe suitable for small short time projects but /tg/station focuses on the long term and thus discourages this.
+### No duplicated code
+Copying code from one place to another may be suitable for small, short-time projects, but /tg/station is a long-term project and highly discourages this.
 
 Instead you can use object orientation, or simply placing repeated code in a function, to obey this specification easily.
 
-### Startup/Runtime tradeoffs with lists and the "hidden" init proc
-First, read the comments in this byond thread, starting here:http://www.byond.com/forum/?post=2086980&page=2#comment19776775
+### Startup/runtime tradeoffs with lists and the "hidden" Init proc
+First, read the comments in [this BYOND thread](http://www.byond.com/forum/?post=2086980&page=2#comment19776775), starting where the link takes you.
 
 There are two key points here:
 
-1) Defining a list in the type definition incurs a hidden proc call - init, if you must define a list at startup, do so in New()/Initialize and avoid the overhead of a second call (init() and then new())
+1) Defining a list in the variable's definition calls a hidden proc - Init. If you have to define a list at startup, do so in New() (or preferably Initialize()) and avoid the overhead of a second call (Init() and then New())
```

Comment:
Which is it in the profiler?
```init()``` or ```Init()``` since I know he named it after people told him just seeing " " sucked.